packages/octez-client/octez-client.21.0/opam

```diff
@@ -0,0 +1,79 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: ["Tezos devteam"]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "3.11.1" }
+  "ocaml" { >= "4.14" }
+  "octez-libs" { = version }
+  "octez-shell-libs" { = version }
+  "uri" { >= "3.1.0" }
+  "octez-protocol-020-PsParisC-libs" { = version }
+  "octez-protocol-021-PsQuebec-libs" { = version }
+]
+depopts: [
+  "tezos-client-genesis"
+  "tezos-client-demo-counter"
+  "octez-protocol-000-Ps9mPmXa-libs"
+  "octez-protocol-001-PtCJ7pwo-libs"
+  "octez-protocol-002-PsYLVpVv-libs"
+  "octez-protocol-003-PsddFKi3-libs"
+  "octez-protocol-004-Pt24m4xi-libs"
+  "octez-protocol-005-PsBabyM1-libs"
+  "octez-protocol-006-PsCARTHA-libs"
+  "octez-protocol-007-PsDELPH1-libs"
+  "octez-protocol-008-PtEdo2Zk-libs"
+  "octez-protocol-009-PsFLoren-libs"
+  "octez-protocol-010-PtGRANAD-libs"
+  "octez-protocol-011-PtHangz2-libs"
+  "octez-protocol-012-Psithaca-libs"
+  "octez-protocol-013-PtJakart-libs"
+  "octez-protocol-014-PtKathma-libs"
+  "octez-protocol-015-PtLimaPt-libs"
+  "octez-protocol-016-PtMumbai-libs"
+  "octez-protocol-017-PtNairob-libs"
+  "octez-protocol-018-Proxford-libs"
+  "octez-protocol-019-PtParisB-libs"
+  "octez-protocol-alpha-libs"
+]
+conflicts: [
+  "tezos-client-genesis" { != version }
+  "tezos-client-demo-counter" { != version }
+  "octez-protocol-000-Ps9mPmXa-libs" { != version }
```

Comment:
It is easier also for the solver is these `!= version` would be replaced by `=version` in the dependency list itself